guideline.md

```diff
@@ -0,0 +1,158 @@
+- Personal expenses or any activity not related to PyLadies.
+- It should be avoided that the opportunities funded benefit repeatedly only a single individual or small group of the chapter members within a single year
+
+## Accessing funds
+
+Chapters wishing to apply for funds should:
+- Open an issue in this repo, choosing the relevant ISSUE type and follow the outlined issue template, this will include a brief statement of the intended use of the funds, detailing the purpose and amount requested. You will also be prompted to add labels relevant to your request
+- If necessary wait for approval (see process below)
+- Submit an invoice directly to the PySV (invoice@python-verband.org) with the pyladies@python-verband.org in CC.
+  - Ensure that all expenditures are accompanied by a proper receipt. 
+  - For major expenditures that include sales tax, the PySV's address (see appendix) must be on the receipt.   
+  - If you are paying a speaker or service provider, an invoice is preferred. However, a signed paper receipt is also acceptable.
+- Reimbursement/ invoice payments will happen within 14 days
+- Move the ticket to reimbursed on our project board adding any additional context regarding the funding we can share in our report (i.e. number of attendees to an event/ one or two outcomes the funding helped you achieve)
+
+At any time you can check the process of requests on our [project board](https://github.com/orgs/PyLadiesGermany/projects/2).
+
+## Approval Process
+
+We want the approval process to be as quick as possible to reduce the load for individual organizers, at the same time we want to ensure that the funds are distributed in a way that maximizes our mission across Germany.
+
+Approvals will be given by the approval group following the approval guidelines (see below). Requests will fall into one of the following approval categories:
+
+### No approval required
+
+For some things we want to ensure chapters can get the quickest access possible, this means the requirement for pre-approval is removed for eligible requests. The chapter must still create a ticket on our platform with relevant information but can immediately send an invoice to the PySV (with the PyLadies email copied in) These items include:
+- Up to 200 EUR for stickers per chapter per year
+- A roll-up banner (one-time purchase) for a chapter
+- Shared subscriptions i.e. streaming software or password managers (these subscriptions should have been agreed upon by the chapters beforehand, you can find a list of shared subscriptions [here](Docs/shared_tools.md))
+
+These quick approvals apply to all existing and future chapters who have had at least one official event under the PyLadies brand or are attending PyCon DE as a group. Stickers for chapters might be ordered together (to reduce costs) and this should be organised between the chapters.
```

Comment:
I think we should drop the future here, so there is no concern that folks not known to the community spend money from the fund without some discussion happening before hand